torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1498,8 +1498,13 @@ at::Tensor& AtenXlaType::leaky_relu_(at::Tensor& self,
 
 at::Tensor AtenXlaType::leaky_relu_backward(const at::Tensor& grad_output,
                                             const at::Tensor& self,
-                                            at::Scalar negative_slope) {
+                                            at::Scalar negative_slope,
+                                            bool self_is_result) {
   XLA_FN_COUNTER("xla::");
+  XLA_CHECK(!self_is_result || negative_slope.to<double>() > 0.0)
```

Comment:
You know my thought in chaing pytorch messages 😉

Reply:
Haha I know, but I feel this message is useful to tell users what happened and what to do? :P